__tests__/app.test.js

```diff
@@ -250,4 +249,36 @@ describe("app tests", () => {
       })
     })
   })
+
+  describe("task 11 get comment count by review id", () => {
+    test("status 200", () => {
+      return request(app).get("/api/reviews/6").expect(200);
+    })
+    test("review data includes comment_count", () => {
+      return request(app).get("/api/reviews/6").expect(200).then(({body}) => {
+        expect(body).toEqual({
+          review_id: 6,
+          title: 'Occaecat consequat officia in quis commodo.',
+          category: 'social deduction',
+          designer: 'Ollie Tabooger',
+          owner: 'mallionaire',
+          review_body: 'Fugiat fugiat enim officia laborum quis. Aliquip laboris non nulla nostrud magna exercitation in ullamco aute laborum cillum nisi sint. Culpa excepteur aute cillum minim magna fugiat culpa adipisicing eiusmod laborum ipsum fugiat quis. Mollit consectetur amet sunt ex amet tempor magna consequat dolore cillum adipisicing. Proident est sunt amet ipsum magna proident fugiat deserunt mollit officia magna ea pariatur. Ullamco proident in nostrud pariatur. Minim consequat pariatur id pariatur adipisicing.',
+          review_img_url: 'https://images.pexels.com/photos/207924/pexels-photo-207924.jpeg?w=700&h=700',
+          created_at: '2020-09-13T14:19:28.077Z',
+          votes: 8,
+          comment_count: '0'
```

Comment:
looks like comment count is a string; is that what you intend? Something to be mindful of